README.md

```diff
@@ -1,24 +1,24 @@
 # koajs-nocache
 
-[![Build Status](https://travis-ci.org/mingchen/koajs-nocache.svg?branch=master)](https://travis-ci.org/mingchen/koajs-nocache)
-
-[![NPM](https://nodei.co/npm/koajs-nocache.png?downloads=true)](https://nodei.co/npm/koajs-nocache/)
-
-
 ## Introduction
 
-A node `koajs` middleware which add no-cache related headers for all the `koajs` response to disable caches.
+A node `koajs` middleware, based off a fork of <https://github.com/mingchen/koajs-nocache>, which add no-cache related headers for all the `koajs` response to disable caches.
 It is useful for REST API response, add no-cache headers to avoid browsers cache request response. Only `GET` response have no-cache headers by default.
 
 The following headers are added to response header:
 
-    Cache-Control: no-cache, no-store, must-revalidate
+    Cache-Control: no-cache, no-store, must-revalidate, proxy-revalidate
     Expires: 0
     Pragma: no-cache
+    Surrogate-Control: no-store
 
 ## Install
 
-    npm install koajs-nocache
+    npm install koajs-nocache@https://github.com/TechtonicGroup/koajs-nocache.git
```

Comment:
does npm now support git paths?

Reply:
@Eric-Dunaway Yes it does https://docs.npmjs.com/cli/install. I also tested it and confirmed it's working